### ISSUE TITLE
[Idea] Give Tyrants back armor

### ISSUE THREAD

**Issue by Gireen** [Unvanquished/gameplay#26]
Related Problem issues: #15

Give tyrants a armored back that reduces damage by 30%
As the opposite to humans where the back is less armored. 

**Disclaimer**
>Ideas without a related problem issue and lacking explanation or depth are closed after a certain time.

**Comment by illwieckz** [comment]
I not only like the idea gameplay wise, I also like it on the lore side of it.

**Comment by Viech** [comment]
That's a fine buff but it does not solve the problem that the Tyrant has a lackluster principal design. It needs _features_, not (just) _stats_.

**Comment by Gireen** [comment]
Well it solves or improves the situation with the related problem.
The tyrant being big and easy to hit. A back armor makes the escape easier but forces players to expose the weaker front if they want to attack. 

Different problems and ideas should be discussed in dedicated idea/problem issues if i may remember you on your own words 😉

**Comment by Viech** [comment]
Fair enough, I did not look up the referenced issue and thought it was "Tyrant as a class is useless". (Which to my defense is a mantra at this point.)

**Comment by bmorel** [comment]
Sorry to disappoint you :)
No, I don't think tyrant is useless: it makes a decent door :D 
More seriously, what I wanted to do with linked issue is to provide the actual numbers, to hopefully be able to have a discussion with good starting bases on how to try to improve it (yet I do not propose anything, because really, I'm unsure about how to improve on it)

**Comment by superdupont-fr** [comment]
Why not an advanced Tyrant with this armor? (and, slower to move too?)
